crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -85,6 +86,8 @@ pub struct TxPool<T: TransactionOrdering> {
     basefee_pool: ParkedPool<BasefeeOrd<T::Transaction>>,
     /// All transactions in the pool.
     all_transactions: AllTransactions<T::Transaction>,
+    /// Transaction pool metrics
+    metrics: TxPoolMetrics,
```

Comment:
can this not live on the outer level in `PoolInner`?

Reply:
(why?)